src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -76,27 +76,27 @@
     }
 
     /**
-     * Parses a {@code Optional<String> phone} into an {@code Optional<Phone>} if {@code phone} is present.
+     * Parses a {@code Optional<String> priority} into an {@code Optional<Priority>} if {@code priority} is present.
      */
-    public static Optional<Priority> parsePhone(Optional<String> phone) throws IllegalValueException {
-        assert phone != null;
-        return phone.isPresent() ? Optional.of(new Priority(phone.get())) : Optional.empty();
+    public static Optional<Priority> parsePriority(Optional<String> priority) throws IllegalValueException {
+        assert priority != null;
+        return priority.isPresent() ? Optional.of(new Priority(priority.get())) : Optional.empty();
```

Comment:
This actually isn't very clean, but it will do for now. Refer to the link I shared on Whatsapp on how to use `isPresent` properly.

Cleaning this up is at the bottom of our priority list so it's fine, just take note the next time :)